types/react-dom/index.d.ts

```diff
@@ -68,3 +68,7 @@ export function unstable_renderSubtreeIntoContainer<P>(
     element: ReactElement<P>,
     container: Element,
     callback?: (component?: Component<P, ComponentState> | Element) => any): Component<P, ComponentState> | Element | void;
+export function unstable_createPortal(
+    children: ReactNode,
+    container: Element,
+    key?: string): ReactElement<any>;
```

Comment:
Is it possible to extend the `ReactNode` type here to include a `ReactPortal` interface that is only defined in the `react-dom` package? I don't think it's right to type this as `ReactElement`.

At the very least, I'd prefer `{}` to `any` here.